README.md

```diff
@@ -14,11 +14,42 @@ Just pass a `loader` method which return a async module import:
 <Loadable loader={() => import('./AsyncComponent.svelte')} />
 ```
 
+Use `unloader` to prevent Loadable from caching the component which will cause it to call `loader` each time the component is used.
+
+```html
+<script>
+  import Loadable from 'svelte-loadable'
+
+  // unloader callback
+  function unloader() {
+    // some code here
+  }
+</script>
+
+<!-- unloader as boolean -->
+<Loadable ... unloader />
+<Loadable ... unloader={true} />
+<Loadable ... unloader={someBooleanValue} />
+
+<!-- unloader as predefined function in script tag above -->
+<Loadable ... {unloader} />
+<!-- unloader as an inline function -->
+<Loadable ... unloader={() => { /* some code here */ }} />
+
+<!-- example using SystemJS Module Loader which has the ability to unload (delete) a previously loaded module -->
+<Loadable
+  loader={() => System.import('./AsyncComponent.svelte')}
+  unloader={() => System.delete(System.resolve('./AsyncComponent.svelte'))}
+/>
+
+```
+
 ### Props
 
 - `loader`: a function which `import()` your component to the `<Loadable>` component.
 - `delay`: minimum delay in `msecs` for showing the `loading slot`. Default: 200
 - `timeout`: time in `msecs` for showing the `timeout slot`.
+- `unloader`: true to prevent the component from being cached or a function which will also prevent the component from being cached and will be called immediatly after it is uncached.
```

Comment:
```suggestion
- `unloader`: `true` to prevent the component from being cached or a `function` which will also prevent the component from being cached after being unmounted and will be called immediately after it is removed from cache.
```